source/administration/migrating.rst

```diff
@@ -48,13 +48,13 @@ This feature can be accessed through the `Mattermost Web App <https://docs.matte
    If you don't use Docker, there are `step-by-step instructions <https://docs.mattermost.com/install/docker-local-machine.html>`_ to install Mattermost in preview mode in less than 5 minutes.
 
 Supported Features
-++++++++++++++++++
+~~~~~~~~~~~~~~~~~~
 
 The following key features are supported when importing from Slack:
 
 * User accounts with an email address set
 
-* Public channels and the text messages posted in them
+* Public channels and the text messages posted in them, with formatting
```

Comment:
My understanding from reading the comments on [PLT-495 Improves slack markup conversion #4914](https://github.com/mattermost/platform/pull/4914) is that all formatting works, but @esethna did the testing and can confirm one way or the other.